src/snapred/backend/dao/StateConfig.py

```diff
@@ -2,29 +2,22 @@
 
 from pydantic import BaseModel
 
+from snapred.backend.dao.calibration import Calibration
 from snapred.backend.dao.state.DiffractionCalibrant import DiffractionCalibrant
 from snapred.backend.dao.state.FocusGroup import FocusGroup
 from snapred.backend.dao.state.NormalizationCalibrant import NormalizationCalibrant
 
 
 class StateConfig(BaseModel):
-    diffractionCalibrant: DiffractionCalibrant
-    emptyInstrumentRunNumber: int
-    normalizationCalibrant: NormalizationCalibrant
+    calibration: Calibration
+    diffractionCalibrant: Optional[DiffractionCalibrant]
+    normalizationCalibrant: Optional[NormalizationCalibrant]
```

Comment:
Why are these being made optional?

Reply:
They're being depreciated as their [contents rely on the prototype caliblog file](https://github.com/neutrons/SNAPRed/blob/next/src/snapred/backend/data/LocalDataService.py#L133-L167), and should not be used going forward. I didnt use the depreciation annotation because that for some reason throws an error.  You'll find we pull/generate their contents elsewhere, a formal mapping may be in order before full removal.

Reply:
I.e DiffractionCalibration refers to data generated via out DiffractionCalibration flow, a `Calibration` and is `CalibrationRecord`. Same for NormalizationCalibrant.